airflow/jobs/scheduler_job_runner.py

```diff
@@ -1817,17 +1828,59 @@ def _fail_tasks_stuck_in_queued(self, session: Session = NEW_SESSION) -> None:
                         )
                         session.add(
                             Log(
-                                event="stuck in queued",
+                                event=RETRY_STUCK_IN_QUEUED_EVENT,
                                 task_instance=ti.key,
                                 extra=(
-                                    "Task will be marked as failed. If the task instance has "
+                                    f"Task was stuck in queued and will be retried, once it has hit {num_allowed_retries} attempts"
```

Comment:
```suggestion
                                    f"Task was stuck in queued and will be requeued, once it has hit {num_allowed_retries} attempts"
```

It might be good to avoid "retry" here to avoid confusion between dag-author-configured-retries and these retries.